packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -243,41 +264,52 @@ module.exports = {
           // in development "style" loader enables hot editing of CSS.
           // By default we support CSS Modules with the extension .module.css
           {
-            test: /\.css$/,
-            exclude: /\.module\.css$/,
-            use: [
-              require.resolve('style-loader'),
-              {
-                loader: require.resolve('css-loader'),
-                options: {
-                  importLoaders: 1,
-                },
-              },
+            test: CSSRegex,
+            exclude: CSSModuleRegex,
+            use: getStyleLoader(
               {
-                loader: require.resolve('postcss-loader'),
-                options: postCSSLoaderOptions,
+                importLoaders: 1,
               },
-            ],
+              'postcss-loader',
+              postCSSLoaderOptions
+            ),
           },
           // Adds support for CSS Modules (https://github.com/css-modules/css-modules)
           // using the extension .module.css
           {
-            test: /\.module\.css$/,
-            use: [
-              require.resolve('style-loader'),
+            test: CSSModuleRegex,
+            use: getStyleLoader(
               {
-                loader: require.resolve('css-loader'),
-                options: {
-                  importLoaders: 1,
-                  modules: true,
-                  getLocalIdent: getCSSModuleLocalIdent,
-                },
+                importLoaders: 1,
+                modules: true,
+                getLocalIdent: getCSSModuleLocalIdent,
               },
+              'postcss-loader',
+              postCSSLoaderOptions
+            ),
+          },
+          // Opt-in support for SASS (using .scss or .sass extensions).
+          // Chains the sass-loader with the css-loader and the style-loader
+          // to immediately apply all styles to the DOM.
+          // By default we support SASS Modules with the
+          // extensions .module.scss or .module.sass
+          {
+            test: SASSRegex,
+            exclude: SASSModuleRegex,
+            use: getStyleLoader({}, 'sass-loader', {}),
```

Comment:
This function call with the two empty objects looks awkward. Can you refactor `getStyleLoader` so that it takes the name of the loader first, loader options second and then `css-loader` options last? Then this call would just be `getStyleLoader('sass-loader')`.

Reply:
Good point, I've changed the order and defaulted the last to params to empty objects.